### PR TITLE
bashdb: 4.4-0.94 -> 4.4-1.0.0

### PR DESCRIPTION
**Pull request by nthorne**
Updated to latest version.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
How do I test this, when I run:

```
$ echo 'echo foo' > foo.sh
$ bashdb ./foo.sh
```

it just hangs

**Comment by nthorne** [comment]
@Mic92 really? Given a file `test.sh` containing

    #!/usr/bin/env bash
    echo "One"
    echo "Two"
    echo "Three"

I get

    $ ./result/bin/bashdb test.sh
    bash debugger, bashdb, release 4.4-1.0.0
    
    Copyright 2002-2004, 2006-2012, 2014, 2016-2018 Rocky Bernstein
    This is free software, covered by the GNU General Public License, and you are
    welcome to change it and/or distribute copies of it under certain conditions.

    ....
     
    (/home/nthorne/repos/nixpkgs/pkgs/development/tools/misc/bashdb/test.sh:3):
    3:      echo "One"
    bashdb<0> n
    One
    (/home/nthorne/repos/nixpkgs/pkgs/development/tools/misc/bashdb/test.sh:4):
    4:      echo "Two"
    bashdb<1> n
    Two
    (/home/nthorne/repos/nixpkgs/pkgs/development/tools/misc/bashdb/test.sh:5):
    5:      echo "Three"
    bashdb<2> n
    Three
    (/home/nthorne/repos/nixpkgs/pkgs/development/tools/misc/bashdb/result/bin/bashdb:1):
    1:      #!/nix/store/r47p5pzx52m3n34vdgqpk5rvqgm0m24m-bash-4.4-p23/bin/bash
    bashdb<3> q
    bashdb: That's all, folks...


so I can step through the source code as expected.

**Comment by nthorne** [comment]
.. a shell script with the shebang only  should at least give you the copyright blurb.. What do you get if you run

    $ bash -x bashdb ./foo.sh

**Comment by Mic92** [comment]
I had to also bring `python3.pkgs.pygments` in my nix-shell scope for this to work. Is this a hard dependency?

**Comment by nthorne** [comment]
Really? I don't have that on my system afaik, and I did a sandbox build when updating the derivation, so I wonder why i had no issues.. I'll look into it. Thanks for the thorough testing!

**Comment by nthorne** [comment]
@Mic92 nope; ran in a pure shell, and works just fine for me. Here's how I tested; please let me know if I'm doing something wrong here:

    $ nix-build <NIXPKGS PATH> -A bashdb
    $ nix-shell -E "with import <nixpkgs> {}; callPackage $PWD/default.nix {}"
    $ cd result/bin
    $ ./bashdb -c ls

I can't figure out why you end up requiring pygments whereas I'm not.. I do see that there's a -S option (which is not working for me) that will pull in the python parts for highlighting support, so perhaps I should go ahead and add it as a `propagatedBuildInputs` anyways..

File: pkgs/development/tools/misc/bashdb/default.nix [hunk 1]
```diff
@@ -1,13 +1,17 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, python36Packages }:
```
Comment (lheckemann) [inline pkgs/development/tools/misc/bashdb/default.nix]: Is this particularly tied to python 3.6? If not, I'd suggest using `python3Packages` here.

Comment (Mic92) [reply pkgs/development/tools/misc/bashdb/default.nix]: Agreed.

Comment (nthorne) [reply pkgs/development/tools/misc/bashdb/default.nix]: Good idea. Fixed that.

**Pull request merged**